src/components/TodoList/TodoList.tsx

```diff
@@ -2,0 +1,15 @@
+import React from 'react';
+import { Todo } from '../../types/todo';
+import { TodoInfo } from '../TodoInfo';
+
+interface TodListProps {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC<TodListProps> = ({ todos }) => (
```

Comment:
The component is using the incorrect interface `TodListProps`. It should be updated to `TodoListProps` after correcting the typo in the interface name.